packages/example/src/pages/guides/v3-migration.mdx

```diff
@@ -28,13 +28,13 @@ built with the Carbon Gatsby theme might encounter.
      [`wrap-root-element.js`](https://github.com/carbon-design-system/gatsby-theme-carbon/blob/857abf691590d86b9384a8f35256e8f54fad299e/packages/gatsby-theme-carbon/src/util/wrap-root-element.js#L1)
      to add back the `ThemeProvider` wrapper.
```

Comment:
I suppose I was assuming that many/most had not migrated to v2.

Reply:
Ah, yeah I'm not sure. If that's the case, I'd vote for keeping the v2 migration doc, and adding a new v3 migration doc. Just in case folks need to go all the way from v1.